app/graphql/types/query_type.rb

```diff
@@ -39,4 +39,14 @@ def primary_location
       Location.primary
     end
   end
+  
+  #Define a field
+  field :schema_sha, String do
+    description "Get current sha"
+  end
+
+  def schema_sha
+    ENV['HEROKU_SLUG_COMMIT']
```

Comment:
I think we've switched this to `AppEnv`

see https://github.com/mojotech/helios2/pull/224